Emlab project section iii

kangsg9790 opened the pull request:
1. BasePoint 따는거 함수 파일 생성

2. DutyCycle 돌리고 따는거 함수 파일 생성

3. 각 함수 출력값 struct화, 각 해석에 필요한 기본 설정 추가

kangsg9790 committed:
1. BasePoint 따는거 함수 파일 생성
2. DutyCycle 돌리고 따는거 함수 파일 생성

kangsg9790 committed:
함수 출력 struct 구조 정리,  각 함수 기본 셋팅 내용 포함

kangsg9790 committed:
Merge commit '97650c99d9d2e2d35c066f66eda97637e05e451e' into emlabProject_SectionIII

enjoyneer87 reviewed (approved):
수고했습니다. 선 merge 후 수정

enjoyneer87 committed:
Update and rename MotorCAD_Lab_BasePoint.m to calcMotorCADLabBasePoint.m

enjoyneer87 committed:
Update and rename MotorCAD_Lab_DutyCycle.m to calcMotorCADLabDutyCycle

함수명을 camelCase 표기로 수정하겠습니다. 관습을 좋아하지는 않지만 이유는 관습입니다.
https://blog.naver.com/ege1001/220466932974
